server/utils/prefetchTags.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+import React from 'react';
+import webpackConfig from '../../webpack.config';
+
+const PUBLIC_PATH = webpackConfig.output.publicPath;
+
+const prefetchTags = [
+  <link rel="dns-prefetch" href={process.env.AWS_S3_UPLOAD_BUCKET_URL} />,
+];
+
+if (process.env.NODE_ENV === 'production') {
+  try {
+    const manifest = require('../../dist/manifest.json');
```

Comment:
How about using the node API's to load this file rather than require - we can just have it output empty if the file isn't found and it'll avoid the env check. That way we'll be able to test it too in the future if we want by mocking a manifest.

Reply:
I googled this a bit; apparently Node doesn't cache the reads so it will always get it from disk. This seemed a faster way to implement the call.

Reply:
True node doesn't but it'd be easy enough to keep it in a local const?

Reply:
true, I'll swap